apps/browser/src/autofill/background/overlay.background.ts

```diff
@@ -1898,11 +1899,24 @@ export class OverlayBackground implements OverlayBackgroundInterface {
     const cardView = new CardView();
     cardView.cardholderName = card.cardholderName || "";
     cardView.number = card.number || "";
-    cardView.expMonth = card.expirationMonth || "";
-    cardView.expYear = card.expirationYear || "";
     cardView.code = card.cvv || "";
     cardView.brand = card.number ? CardView.getCardBrandByPatterns(card.number) : "";
 
+    // If there's a combined expiration date value and no individual month or year values,
+    // try to parse them from the combined value
+    if (
+      card.expirationDate &&
+      (card.expirationMonth == null || card.expirationMonth === "") &&
+      (card.expirationYear == null || card.expirationYear === "")
```

Comment:
```suggestion
      !card.expirationMonth &&
      !card.expirationYear
```

❓ - Would any issues be experienced by using falsy coercion here?

Reply:
This was meant to be defensive against a case where either value is `0`...

but, `0` isn't a valid year value (and so could be considered empty), we store months as zero-indexed values, and these CardView properties are only stored as strings (afaik) anyway. As long as none of that changes (an if it does, we'll have other downstream work to do), we don't actually need this level of value protection. 👍

Reply:
^ one note: we don't actually validate user input for the card inputs; we rely on the input type constraints, but will store and return invalid strings, so we still need to cover those cases.